packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -165,7 +170,12 @@ export function Gauge(props: GaugeProps) {
       {description && isHovering ? (
         <Box className={classes.description}>{description}</Box>
       ) : (
-        <Box className={classes.overlay}>
+        <Box
+          className={classNames(
+            classes.overlay,
+            size === 'small' ? classes.overlaySmall : undefined,
```

Comment:
non-blocking, but `classNames` should also support smth. like you used in `InfoCard`: `[classes.contentAlignBottom]: alignContent === 'bottom'`.

Reply:
Fixed this, the object property version is a lot nicer, thanks!